lib/prorate/throttled.rb

```diff
@@ -0,0 +1,8 @@
+class Prorate::Throttled < StandardError
```

Comment:
The name of this file could be better, as now it's not clear this is an exception.

Reply:
It could but it follows the convention of the `filename -> module`, and the module is a subclass of Exception which we expose externally as part of the API. So a compromise here.